site/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState, FC, ReactNode, PropsWithChildren } from "react"
+import Collapse from "@mui/material/Collapse"
+// eslint-disable-next-line no-restricted-imports -- It is the base component
+import MuiAlert, { AlertProps as MuiAlertProps } from "@mui/material/Alert"
+import Button from "@mui/material/Button"
+
+export interface AlertProps extends PropsWithChildren {
+  severity: MuiAlertProps["severity"]
```

Comment:
If `ErrorAlert` is going to be a separate component, should we omit the severity `error` from this prop type? At first glance, it's hard to tell if I should be using `ErrorAlert` for errors, or `Alert` with `severity="error"` or both in different ways.

Reply:
`Alert` is the generic one with extra things and it should still be usable for errors if you need some customization. The advantage of using the `ErrorAlert` component is, you can pass the error and get the message and detail displayed without having to do it yourself.